make/langtools/src/classes/build/tools/symbolgenerator/CreateSymbols.java

```diff
@@ -338,6 +338,10 @@ public void createSymbols(String ctDescriptionFileExtra, String ctDescriptionFil
             "Ljdk/internal/PreviewFeature;";
     private static final String PREVIEW_FEATURE_ANNOTATION_NEW =
             "Ljdk/internal/javac/PreviewFeature;";
+    private static final String RESTRICTED_ANNOTATION =
+            "Ljdk/internal/javac/Restricted;";
+    private static final String RESTRICTED_ANNOTATION_INTERNAL =
```

Comment:
These should be moved around, now we're intermixing preview-related decls with restricted ones